fhd_core/fhd_main.pro

```diff
@@ -253,8 +253,11 @@ IF Keyword_Set(export_images) THEN BEGIN
 ENDIF
 
 ;optionally export frequency-splt Healpix cubes
+;;; Temporarily --- Include options from simulation to select a circular region of healpix indices
```

Comment:
Is this temporary choice is relevant?

Reply:
I think this was me trying to start including these options in the FHD code for data analysis. It has no effect on simulations, so I'll just remove the change.